### ISSUE TITLE
There are multiple flashes on the page, but when one of them cannot be played, the other normally played ones will also be given unplayable information

### ISSUE THREAD

**Issue by wangstar** [ruffle-rs/ruffle#12170]
### Describe the bug

![flash](https://github.com/ruffle-rs/ruffle/assets/4952511/5ae6d707-3524-4459-bf8e-c8051e51fb76)

### Expected behavior

Error messages should only be displayed on unsupported ruffle-player

### Affected platform

Browser's extension

### Operating system

windows10

### Browser

chrome

**Comment by n0samu** [comment]
This is a known problem - currently if one Ruffle instance on a page panics, so do all of the others.

**Comment by wangstar** [comment]
> This is a known problem - currently if one Ruffle instance on a page panics, so do all of the others.

Are there plans to fix this?

**Comment by n0samu** [comment]
I don't think so, because sufficiently separating Ruffle instances might increase resource usage too much.
